src/index.html

```diff
@@ -9,6 +9,129 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <h1>Human rights</h1>
+  <h1 class="mainTitle">Universal Declaration of Human Rights</h1>
+  <h2 class="secondaryTitle">Preamble</h2>
+    <p class="regularText">Whereas recognition of the inherent dignity and of the equal and inalienable
```

Comment:
why `p` has extra indentations? it's not a nested tag into `h2`;
when you write some text inside tags move it to the new line and add proper indentations so that the code looks clear